sql/core/src/test/resources/tpcds-query-results/v1_4/q77.sql.out

```diff
@@ -4,8 +4,8 @@
 struct<channel:string,id:int,sales:decimal(27,2),returns:decimal(27,2),profit:decimal(28,2)>
 -- !query output
 NULL	NULL	238379361.39	11949589.80	-69066318.65
-catalog channel	NULL	116209.49	1989207.49	-1103184.43
```

Comment:
`RDD.takeOrdered` and `RDD.top` are used in many place.
In SQL, it's used in [TakeOrderedAndProjectExec](https://github.com/apache/spark/blob/d1ca0ea4ae55bf19c7569eacae5b377c3340dde9/sql/core/src/main/scala/org/apache/spark/sql/execution/limit.scala#L285-L300), to pass the `TPCDS querys`, I had to swap two line in the results of `q77` and `q77a`. the [`q77`](https://github.com/apache/spark/blob/a2d8d767d933321426a4eb9df1583e017722d7d6/sql/core/src/test/resources/tpcds/q77.sql#L99-L100) and [`q77a`](https://github.com/apache/spark/blob/5f653d4f7c84e6147cd323cd650da65e0381ebe8/sql/core/src/test/resources/tpcds-v2.7.0/q77a.sql#L119-L121) both end with `order by channel, id limit 100`, and the swapped lines have the same `(channel, id)`, so I guess this change is fine.

But I think it still need you to take a look @cloud-fan @HyukjinKwon

Reply:
this is fine